rasa_core_sdk/executor.py

```diff
@@ -24,8 +24,19 @@ def __init__(self):
 
         self.messages = []
 
+    # deprecated
     def utter_custom_message(self, *elements):
         # type: (*Dict[Text, Any]) -> None
+
+        logger.warning(
```

Comment:
the deprecation is a great idea :+1:

let's use the warnings module though, as that will allow users to ignore it if they so please:
```python
warnings.warn(
    "Use of `utter_custom_message` is deprecated. "
    "Use `utter_elements` to send elements, or "
    "`utter_custom_json` to send a custom json message. ",
    DeprecationWarning,
)
```